### ISSUE TITLE
Setting attention mask in bert-large-uncased-whole-word-masking-finetuned-squad

### ISSUE THREAD

**Issue by svanschalkwyk** [explosion/spacy-transformers#150]
Trying to use SpaCy for SQUAD. 
```
name = "bert-large-uncased-whole-word-masking-finetuned-squad"
nlp = TransformersLanguage(trf_name=name, meta={"lang": "en"})
nlp.add_pipe(nlp.create_pipe("sentencizer"))
nlp.add_pipe(TransformersWordPiecer.from_pretrained(nlp.vocab, path))
nlp.add_pipe(TransformersTok2Vec.from_pretrained(nlp.vocab, path))
```
qa = nlp(question....
[and I need help further down as well, i.e. logits]
Any help would be greatly appreciated.

**Comment by svlandeg** [comment]
Hi!

I'm not sure what you're trying to do exactly. If you want general coding help, a better forum is probably StackOverflow. If you run into specific issues with this library, a reproducible code snippet is useful to help us debug/understand your issue. 

FYI - you may have already noticed, but we recently released the `v1.0.0rc0` version of this library, which is compatible with the pre-release of spaCy 3, cf. [`spacy-nightly`](https://pypi.org/project/spacy-nightly/). I think this library is much more powerful and robust now, and we've also written quite some documentation, see for instance https://nightly.spacy.io/usage/embeddings-transformers#transformers and https://nightly.spacy.io/api/transformer. Hopefully those docs will guide you on your way! Good luck and have fun ;-)
